tests/foreman/ui/test_errata.py

```diff
@@ -535,15 +542,39 @@ def test_positive_content_host_search_type(session, erratatype_vm):
     assert _install_client_package(erratatype_vm, pkgs, errata_applicability=True)
 
     with session:
+        # Search for RHSA security errata
         ch_erratum = session.contenthost.search_errata(
             erratatype_vm.hostname, "type = security", environment='Library Synced Content'
         )
 
+        # Assert length matches known amount of RHSA errata
         assert len(ch_erratum) == FAKE_9_YUM_SECURITY_ERRATUM_COUNT
 
-        # check IDs just to be safe
+        # Assert IDs are that of RHSA errata
         errata_ids = sorted([e['Id'] for e in ch_erratum])
         assert errata_ids == sorted(FAKE_9_YUM_SECURITY_ERRATUM)
+        # Search for RHBA buxfix errata
+        ch_erratum = session.contenthost.search_errata(
+            erratatype_vm.hostname, "type = bugfix", environment='Library Synced Content'
+        )
+
+        # Assert length matches known amount of RHBA errata
+        assert len(ch_erratum) == FAKE_10_YUM_BUGFIX_ERRATUM_COUNT
+
+        # Assert IDs are that of RHBA errata
+        errata_ids = sorted([e['Id'] for e in ch_erratum])
```

Comment:
instead of `e` we should give some appropriate name 'errata'

Reply:
yep, can do (that is old code)

Reply:
No with:         `errata_ids = sorted([erratum['Id'] for erratum in ch_erratum])`